### PR TITLE
Solution: JS Named vs. anonymous callback functions challenge

### PR DESCRIPTION
**Pull request by yuvaliacob**

### CODE REVIEW

File: js/JS-Callback-Functions/js-callback-functions_user-login-callback-1-named-vs-anonymous/js/index.js [hunk 1]
```diff
@@ -5,4 +5,12 @@ function handleUserLogin(onSuccess) {
...
 }
 
-// Call handleUserLogin below!
+function showWelcomeMessage() {
+  console.log("Welcome! You are logged in now.");
+}
+
+handleUserLogin(showWelcomeMessage);
+
+handleUserLogin(() => {
```
Comment (jamoragar) [inline js/JS-Callback-Functions/js-callback-functions_user-login-callback-1-named-vs-anonymous/js/index.js:14]: This extra line is no needed at all

**Review by jamoragar — APPROVED** [review]
Your code works!
But it has an extra functions? check it out 👍🏻

**Comment by jamoragar** [comment]
Sorry! I didn't read the instructions 😅
I thought that would be weird if you have 2 logs in the console.

My fault!

**Comment by yuvaliacob** [comment]
Thanks, @jamoragar!

**Pull request merged**